integration/kubernetes/alluxio-master.yaml

```diff
@@ -0,0 +1,81 @@
+#
```

Comment:
perhaps call this alluxio.master.yaml.template instead since users will often need to update the values in here.